src/main/java/org/toradocu/translator/JavaElementsCollector.java

```diff
@@ -171,7 +171,9 @@ private static ClassCodeElement containingClassOf(DocumentedExecutable documente
                 .map(PropositionSeries::getSemanticGraph)
                 .collect(toList());
         for (SemanticGraph sg : sgs) {
-          ids.add(sg.getFirstRoot().word());
+          if (sg.getFirstRoot() != null) {
```

Comment:
It might help to comment on `sg.getFirstRoot()` returning `null`. Do we expect it? If it happens, can we safely ignore it?